marconi-chain-index/src/Marconi/ChainIndex/Indexers/Utxo.hs

```diff
@@ -1026,33 +1041,40 @@ instance Queryable UtxoHandle where
              GROUP BY s.slotNo
              ORDER BY s.slotNo DESC
              LIMIT ?|]
+        queryTip =
```

Comment:
Should we remove the old chain tips and only keep the most recent one?

Also, I don't see the insert statement of the chainTip. Did I miss it?

Reply:
Yes, I may have fat findgered a late rebase, it was there and it isn't anymore, let me fix this before you continue.